musicFunction.js

```diff
@@ -0,0 +1,52 @@
+var original;
+var i = 0;
+
+window.onload = function() {
+original = document.getElementById('container');
+
+};
+
+function playSound(id){
+	//duplicate 
+	var clone = original.cloneNode(true); // clone
+	clone.id = "container" + ++i; // there can only be one element with an ID
+	original.parentNode.appendChild(clone);
+	//play sound
+	sound(id);
+	
+}
+
+function sound(x){
+	const synth = new Tone.Synth().toMaster();
+	var num = x.slice(9);
+	num = parseInt(num);
+	//play different notes randomly based on number of flower
+	if(num % 2 == 0){
+		synth.triggerAttackRelease("C3", "8n");
+	}
+	else if(num % 3 == 0){
+		synth.triggerAttackRelease("C5", "8n");
+	}
+	else if(num % 4 == 0){
+		synth.triggerAttackRelease("B3", "8n");
+	}
+	else if(num % 5 == 0){
+		synth.triggerAttackRelease("D3", "8n");
+	}
+	else if(num % 6 == 0){
+		synth.triggerAttackRelease("B5", "8n");
+	}
+	else if(num % 7 == 0){
+		synth.triggerAttackRelease("D5", "8n");
+	}
+	else if(num % 8 == 0){
+		synth.triggerAttackRelease("E5", "8n");
+	}
+	else if(num % 9 == 0){
+		synth.triggerAttackRelease("E3", "8n");
+	}
```

Comment:
if a num % 8 === 0, then num % 4 === zero as well so i believe the earlier if statement will be true and you'll probably never hear the 6, 8 or 9 tones. You might be able to write the if else blocks in fewer lines if you used a switch statement. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

Reply:
Oh you are correct! the numbers that are multiples of each other wont make it. Yes, in hind sight I should have used a switch statement haha thank you :)